### ISSUE TITLE
Making the gas fees 0.005 (or 1c)

### ISSUE THREAD

**Issue by spacechicken** [mikealeonetti/drip-auto-hydrater#6]
Hey Mike,

I used the changes that George did and it gives me a 0.01c GAS fee then a 0.25c GAS fee then a 0.01 GAS fee. Will your new code make it slow down the transaction down so all transactions will be 0.005 or 0.1c? 

I have modified it a bit so its a bit of work to add your big update.

Thanks for your assistance.

**Comment by mikealeonetti** [comment]
I've been testing the code I shared with you both which is my current configuration. I get prices between $0.01 to $0.10 per transaction.

If you want to go any lower, you will need to play with the gas and gasPrice settings until you get what you want.

The transactions by design as they get cheaper should be slower. I've noticed that in my testing. But if I'm not trying to ape into a coin, I don't care. It's not like on Polygon network where sometimes it takes half a day to clear a transaction. For me it takes 5 minutes at most. Since it happens automatically I barely even notice.
